### PR TITLE
Feat/add navbar footer and reports page

### LINKED ISSUE #5: [Layout] Implement Navbar

### LINKED ISSUE #6: [Layout] Implement Footer

### PR DESCRIPTION
**Pull request by SpottedYodeler2**
closes #5
closes #6

### CODE REVIEW

**Comment by SpottedYodeler2** [comment]
feat: add navbar #5  footer #6,  reports page #12

File: src/components/AddReport.jsx
Comment (chaaals) [inline src/components/AddReport.jsx:1]: remove if not yet implemented

File: src/layout/index.jsx [hunk 1]
```diff
@@ -1,11 +1,13 @@
 import { Outlet } from "react-router-dom";
+import NavBar from "../components/NavBar";
+import Footer from "../components/Footer";
 
 const Layout = () => {
   return (
     <main className="w-full bg-colorPrimary h-screen py-16 px-24">
-      {/* Add navbar */}
+      <NavBar />
```
Comment (chaaals) [inline src/layout/index.jsx:8]: put outside <main>, wrap with react fragment.
```
<>
<Navbar />
<main></main>
<Footer />
</>
```

File: src/layout/index.jsx [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
       <Outlet />
-      {/* Add footer */}
+      <Footer />
```
Comment (chaaals) [inline src/layout/index.jsx:12]: follow comment above

**Review by uzuriel — APPROVED** [review]

File: src/pages/ReportPage.jsx [hunk 3]
```diff
@@ -0,0 +1,102 @@
+import React, { useState } from "react";
+
+// Header Component
+const Header = () => {
+  return (
+    <header className="flex justify-between items-center p-2 bg-transparent mt-12">
```
Comment (chaaals) [inline src/pages/ReportPage.jsx]: use section instead of header

File: src/pages/ReportPage.jsx [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+        )}
+      </div>
+    </div>
+  );
+};
+
+// ReportPage Component (Combining Everything)
+const ReportPage = () => {
+  return (
+    <div className="flex flex-col min-h-screen bg-transparent">
```
Comment (chaaals) [inline src/pages/ReportPage.jsx]: use section

**Review by chaaals — CHANGES_REQUESTED** [review]

**Comment by SpottedYodeler2** [comment]
@chaaals I applied na your reviews

**Review by chaaals — APPROVED** [review]

**Review by uzuriel — APPROVED** [review]

**Pull request merged**